README.md

```diff
@@ -116,7 +116,7 @@ To fix all Linter error automatically
 ## 🔭 Future Features <a name="future-features"></a>
 
 - [ ] **Desktop version**
-- [ ] **Delete trasnsaction feature**
+- [ ] **Delete transaction feature**
```

Comment:
- [ ] I am giving the comment here because this is the only file changed in the PR you created, kindly move the place you gave credit to the author of the design `Gregoire Vella` to the acknowledgement section and also, always try to open a PrR with the branch with the most part of your code changes